src/Avalonia.DesignerSupport/Remote/HtmlTransport/SimpleWebSocketHttpServer.cs

```diff
@@ -323,8 +323,7 @@ async Task SendFrame(FrameType type, byte[] data, int offset, int length)
                     header.Length64 = (ulong) IPAddress.HostToNetworkOrder((long) length);
                 }
 
-                var endOfMessage = true;
-                header.Mask = (byte) (((endOfMessage ? 1u : 0u) << 7) | ((byte) (type) & 0xf));
+                header.Mask = (byte)(128u | ((byte)type & 0xf));
```

Comment:
It was here for future frame splitting support.

Reply:
I reverted it